ui/src/app/machines/views/MachineDetails/MachineStorage/MachineStorage.tsx

```diff
@@ -21,6 +22,7 @@ const MachineStorage = (): JSX.Element => {
   if (machine && "disks" in machine && "special_filesystems" in machine) {
     return (
       <>
+        <StorageNotifications id={id} />
```

Comment:
I put the summary notifications inside MachineDetails. I was thinking if they're at that level they'll be in a consistent spot and won't accidentally get wrapped in another element, what do you think?

https://github.com/canonical-web-and-design/maas-ui/blob/master/ui/src/app/machines/views/MachineDetails/MachineDetails.tsx#L57

Reply:
Yeah that sounds good!